frontend/views/containers/chatroom/ChatMain.vue

```diff
@@ -454,8 +458,14 @@ export default ({
             beforeRequest
           }
         }).catch((e) => {
-          console.error(`Error while publishing message for ${contractID}`, e)
-          alert(e?.message || e)
+          if (e.cause?.name === 'ChelErrorFetchServerTimeFailed') {
+            alert('Can\'t send message when offline, please connect to the Internet')
```

Comment:
- Missing `L` function
- Recommend using double-quotes to avoid having to escape single-quote

Reply:
Nice catch. I will cover it with `L`. However, I remember we are using double-quotes only when designing vue components

```
...
throw new Error('Username can\'t be deleted')
...
[PROPOSAL_GENERIC]: {
      [PROPOSAL_VARIANTS.CREATED]: L('{from} created a proposal. "{title}"', options),
      [PROPOSAL_VARIANTS.EXPIRING]: L('Proposal from {from} is expiring. "{title}"', options),
      [PROPOSAL_VARIANTS.ACCEPTED]: L('{from}\'s proposal is accepted. "{title}"', options),
      [PROPOSAL_VARIANTS.REJECTED]: L('{from}\'s proposal is rejected. "{title}"', options),
      [PROPOSAL_VARIANTS.EXPIRED]: L('{from}\'s proposal is rejected. "{title}"', options)
}
...
L('{span_}1 member{_span} hasn\'t {strong_} logged in past {days} days or more {_strong}', argsMap['haventLoggedIn'])
```

Reply:
It is fine (preferable) to use double-quotes here in this situation because no double-quotes are being shown to the user, and so in this case we can use double-quotes to avoid having to escape the single-quote that's being shown to the user.